version

```diff
@@ -0,0 +1 @@
+v1.5.4
```

Comment:
The `version` file is auto-generated by the build process. It shouldn't be checked into the repo.